docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-configuring-and-running-an-application-analysis_{context}"]
+= Configuring and running an application analysis
+
+You can analyze more than one application at a time against more than one transformation target in the same analysis.
+
+.Procedure
+
+. In *Development* view, click *Application inventory*.
+. Click the *Analysis* tab.
+. Select the applications you want to analyze.
++
+// ![](/Tackle2/AppAssessAnalyze/SelectManageCred.png)
+
+. Review the credentials assigned to the application.
+. Click *Analyze*.
++
+// ![](/Tackle2/AppAssessAnalyze/AnalysisMode.png)
+
+. Select the *Analysis mode* from the list. Valid options are:
+* Binary.
+* Source code.
+* Source code and dependencies.
+* Upload a local binary. This option only appears if you are analyzing a single aplication.
+
+. If you choose *Upload a local binary*, a window opens and you are prompted to *Upload a local binary*. Either drag and drop a file into the area provided or click *Upload* and then select the file to upload.
+
+. Click *Next*.
+. Select one or more transformation targets:
+
+* Containerization
+* Quarkus
+* OracleJDK to OpenJDK
+** OpenJDK 11
+** OpenJDK 17
+* Linux
+* Camel
+* Azure AppService
+
```

Comment:
The statement for Set Targets "**Select one or more target options in focus for the analysis**."
And also more targets are present now, Please refer the Set targets on latest MTA
![Screenshot from 2022-11-03 20-56-41](https://user-images.githubusercontent.com/87791967/199764255-2a12cc69-a07a-4ab9-994c-1e703729950e.png)

Reply:
@kpunwatk -- I need screen captures or a video -- what are the options in the application server migration to list? What are options when the user clicks Next?

Reply:
Hi @RichardHoch, Thanks, below I've attached the video. If anything you need or have asked, please let me know.
[Screencast from 11-07-2022 12:17:15 PM.webm](https://user-images.githubusercontent.com/87791967/200243167-01c12d70-53b4-48d4-8e87-41f3ec5bbfb7.webm)

Reply:
Right now there are 11 targets with three having sub-targets(with the drop-down options).

Reply:
@kpunwatk -- Thanks. The video helped a lot!

Reply:
For Azure only "Azure App Service" is present